ui/components/src/Legend/ListLegendItem.tsx

```diff
@@ -44,3 +49,5 @@ export const ListLegendItem = React.memo(function ListLegendItem({ item, sx, ...
     </ListItem>
   );
 });
+
+export const ListLegendItem = React.memo(ListLegendItemBase);
```

Comment:
I do the `memo` separately, so we can have a function name associated with the output of the `forwardRef`. Otherwise, eslint complains about the component not having a `displayName`.